Fix #48 Re-enable send_notification tests

mstniy opened the pull request:
The test that tests the happy path is still disabled, because the code doesn't seem to be able to send emails outside KTH (maybe some form of authentication is required?).

Fixes #48

mstniy committed:
Fix #48 Re-enable send_notification tests
The test that tests the happy path is still disabled, because the code doesn't seem to be able to send emails outside KTH (maybe some form of authentication is required?).

philippaornell commented:
Yes, I do think some kind of auth is needed! :) Have a look at this link: https://www.javatpoint.com/example-of-sending-email-using-java-mail-api and see the example of " sending email in Java through SMTP server provided by the host provider". The code is pretty similar to yours but they also use a javax.mail.Authenticator() method. What do you think? :) (It seems quite reasonable that password need to be authenticated).

mstniy committed:
Feat #48 Fix send_notification to use an existing Gmail account

The send_notification success test is still commented out to keep my email from overflowing with spam. Enable it on your working directory if you want to test it.

mstniy commented:
Fixes #52 as well

mackeper reviewed (approved):
Everything seems to work
